.travis.yml

```diff
@@ -21,8 +21,10 @@ install:
 - pip install pygments
 - npm install -g asyncapi-generator@0.6.7 # 0.7 only supports AsyncAPI 2.0
 - npm install -g concat-json-files
+- pip install -r tests/requirements.txt
 
 script:
+- pytest tests
```

Comment:
Thanks for adding a solution for the next time. In the long run I hope this is automatically checked by speccy, see https://github.com/wework/speccy/issues/287

Reply:
I'll add a comment so it can be removed when speccy provides this by default